tests/tests_integration/tests_core_integration/tests_db_writer_integration/test_kafka_writer_integration.py

```diff
@@ -80,19 +82,18 @@ def test_kafka_writer_snapshot(spark, kafka_processing):
     pd_df = processing.get_expected_df(topic, num_messages=df.count(), timeout=3)
 
     assert len(pd_df) == df.count()
-    processing.assert_equal_df(df, other_frame=pd_df.drop(columns=["key", "partition", "headers"], axis=1))
+    processing.assert_equal_df(df, other_frame=pd_df.drop(columns=["key", "partition", "headers", "topic"], axis=1))
```

Comment:
```suggestion
    expected_df = df.withColumn("key", lit(None)).withColumn("headers", lit(None))
    processing.assert_equal_df(pd_df, other_frame=expected_df)
```
И проверки ниже на null будут не нужны